domain/access/doc.go

```diff
@@ -4,4 +4,17 @@
 // Package access provides the services for managing users and permissions in
 // Juju. The users side is mainly concerned with authentication and the
 // permissions side with authorization.
+//
+// External users (i.e. users with an external domain in their user name e.g.
+// someone@external) have their permissions resolved against the
+// everyone@external user.
+//
+// The permissions for everyone@external act like a group for external users.
+// When any external users permissions are read the users access level will be
+// resolved against the access level of everyone@external. If everyone@external
```

Comment:
Knitpick: 'resolved against' is vague. 

I do see you clarify in the next couple of sentences, but this is still could be a bit of a tripping point.

To me,'resolved' gives the impression it has a specific technical definition, meaning I'm supposed to pick up more information than I'm really able to